public/app/features/explore/RichHistory/RichHistoryContainer.test.tsx

```diff
@@ -1,14 +1,24 @@
 import { render } from '@testing-library/react';
 import React from 'react';
 
-import { SortOrder } from '../../../core/utils/richHistoryTypes';
+import { SortOrder } from 'app/core/utils/richHistory';
+
 import { ExploreId } from '../../../types/explore';
 
 import { Tabs } from './RichHistory';
 import { RichHistoryContainer, Props } from './RichHistoryContainer';
 
 jest.mock('../state/selectors', () => ({ getExploreDatasources: jest.fn() }));
 
+jest.mock('@grafana/runtime', () => ({
+  ...(jest.requireActual('@grafana/runtime') as unknown as object),
```

Comment:
```suggestion
  ...jest.requireActual('@grafana/runtime'),
```
same as before, do we need `as unknown as object`?

Reply:
Fixed in 01d3c34eee6438605d49e11f2337147bf4887ae2